modules/nf-core/modules/rootseg/main.nf

```diff
@@ -0,0 +1,34 @@
+// Import generic module functions
```

Comment:
This module should be moved to the `nf-core/local` directory as it is not an nf-core module